src/drop.js

```diff
@@ -23,12 +23,12 @@ export default (config, xScale) => selection => {
         .enter()
         .append('circle')
         .classed('drop', true)
-        .attr('r', dropRadius)
-        .attr('fill', dropColor)
         .on('click', onClick)
         .on('mouseover', onMouseOver)
         .on('mouseout', onMouseOut)
         .merge(drops)
+        .attr('r', dropRadius)
```

Comment:
Can you try add a test to ensure it won't break again? Probably by rendering a chart, updating its data and check both radius and color?

Reply:
@jpetitcolas from the issue #241 it seems that in order to test this, the zoom needs to be invoked on the chart. I don't really know how this can be done externally without writing a function that does it. However, the issue does describe the bug in detail. 

Could maybe add enough data and then somehow zoom in and out, then check the data that was outside the viewable area? Could maybe dispatch a zoom event (the custom dispatch ones) and then reset the zoomIdentity, which I think is doable. But I doubt I have time to mess around with this unfortunately.

Reply:
Indeed ,forgot that testing zoom was not done for now. And not sure on how to do it. Merged! :)